include/rmm/mr/device/thrust_allocator_adaptor.hpp

```diff
@@ -121,9 +121,12 @@ class thrust_allocator : public thrust::device_malloc_allocator<T> {
   }
 
   /**
-   * @briefreturn{The resource used to allocate and deallocate}
```

Comment:
This has just been added in the recent push so I am pretty sure it is safe to rename the method so that it is consistent with the other adaptors